<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prosim</class>
 <widget class="QMainWindow" name="Prosim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Prosim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>794</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="gridFrame">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>3</y>
       <width>788</width>
       <height>368</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="processGraph">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="memoryGraph">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>701</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="gridFrame">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <widget class="QSpinBox" name="expirySpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="algorithmModes">
        <item>
         <property name="text">
          <string>FCFS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SRT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPN</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSpinBox" name="quantumSpinBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="gridFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="algorithmLabel">
        <property name="text">
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="expiryLabel">
        <property name="text">
         <string>Expiry</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="quantumLabel">
        <property name="text">
         <string>Quantum</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="gridFrame">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>-1</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="3" column="0">
       <widget class="QLineEdit" name="outputFileTextBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="simulationModes">
        <item>
         <property name="text">
          <string>P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VM + P</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="inputFileTextBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="inputFileDialogue">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="outputFileDialogue">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="gridFrame">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="simulationModeLabel">
        <property name="text">
         <string>Simulation Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="outputFileLabel">
        <property name="text">
         <string>Output File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="inputFileLabel">
        <property name="text">
         <string>Input File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
